linux_5.10/drivers/remoteproc/cvitek_remoteproc.c

```diff
@@ -145,7 +148,10 @@ static int cvitek_rproc_parse_fw(struct rproc *rproc, const struct firmware *fw)
 		index++;
 	}
 
-	return rproc_elf_load_rsc_table(rproc, fw);
+	if (rproc_elf_load_rsc_table(rproc, fw))
```

Comment:
In my opinion, this is a reasonable check by the kernel's remoteproc framework and should not be bypassed. Instead, I think a `.resource_table` SECTION should be added in FreeRTOS.